.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,91 @@
+# Workflow to run tests
+
+name: auto-merge
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build-and-test:
+    strategy:
+      fail-fast: false
+
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: kotlin-1.5.20
+
+    - name: merge commits from main to release branch
+      run: |
+        # Cherry pick new changes from main, except for version bumps.
+        # A commit is a version bump IFF it touches third_party/prebuilt/repo
+        git config --global user.email "kspautopick@example.com"
```

Comment:
while this is not a real email address, is `@example.com` intentional?

Reply:
Done.